src/strategies/tokenlon/index.ts

```diff
@@ -0,0 +1,54 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'TokenlonDev';
+export const version = '0.1.0';
+
+const TFT_ADDRESS = '0xE534619dEFDBF0cAf673b8AbF7158714F5BD4bd9';
+
+const abi = [
+  {
+    constant: true,
+    inputs: [
+      {
+        internalType: 'address',
+        name: 'account',
+        type: 'address'
+      }
+    ],
+    name: 'balanceOf',
+    outputs: [
+      {
+        internalType: 'uint256',
+        name: '',
+        type: 'uint256'
+      }
+    ],
+    payable: false,
+    stateMutability: 'view',
+    type: 'function'
+  }
+];
+
+export async function strategy(
```

Comment:
Instead of writing again the `erc20-balance-of` strategy you can import it like this: https://github.com/snapshot-labs/snapshot.js/blob/master/src/strategies/erc20-balance-of-coeff/index.ts that will simplify / reduce the code.

Reply:
Done. A big thx again.